_posts/2015-06-17-Estonia.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "What Estonia can teach us about the future of government records"
+date: 2015-06-17
+layout: post
+author:
+- emileigh
+tags:
+- usds
+- event
+excerpt: "At 18F, we were thrilled to meet for a discussion on digital services,
+cyber security, and IT funding with Estonia's most prominent technologists."
+description: "At 18F, we were thrilled to meet for a discussion on digital services,
+cyber security, and IT funding with Estonia's most prominent technologists."
+image: /assets/blog/Estonia/Estonia.jpg
```

Comment:
This should be either `Estonia.jpg` or `Estonia2.jpg` depending on which we keep.